src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,142 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
```

Comment:
Not fixed
![image](https://user-images.githubusercontent.com/20326858/202435014-4009e425-910b-4c78-8f28-88005df4fbba.png)